docs/src/layouts/AppFooter.tsx

```diff
@@ -16,7 +16,13 @@ import ROUTES from 'docs/src/route';
 import Link from 'docs/src/modules/components/Link';
 import SvgStackOverflow from 'docs/src/icons/SvgStackOverflow';
 
-export default function AppFooter() {
+interface AppFooterProps {
+  stackOverflowURL?: string;
```

Comment:
A small detail: we have used this naming convention so far in the public API for acronyms:

```suggestion
  stackOverflowUrl?: string;
```